testsuite/pytests/test_eprop_plasticity.py

```diff
@@ -0,0 +1,738 @@
+# -*- coding: utf-8 -*-
+#
+# test_eprop_plasticity.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Test functionality of e-prop plasticity.
+"""
+
+import nest
+import numpy as np
+import pytest
+
+
+def test_ConnectNeuronsWithEpropSynapse():
```

Comment:
I will soon send a PR with a more pynest'y implementation of this and the next test.